src/scripts/Tara.deploy.s.sol

```diff
@@ -38,7 +38,7 @@ contract TaraDeployer is Script {
         bytes memory _current_sync_committee_aggregated_pubkey = vm.envBytes("AGGREGATED_PUBLIC_KEY");
         bytes32 _genesis_validators_root = 0x9143aa7c615a7f7115e2b6aac319c03529df8242ae705fba9df39b79c59fa8b1;
 
-        BeaconLightClient client = new BeaconLightClient(
+        BeaconLightClient beacon_client = new BeaconLightClient(
```

Comment:
```suggestion
        BeaconLightClient beaconClient = new BeaconLightClient(
```
I've been using `snake_case` when the parameters were coming from the python script only. Let's stick to camelCase for solidity as we discussed.